tests/test_quadratic_discriminant_analysis_converter.py

```diff
@@ -0,0 +1,101 @@
+# SPDX-License-Identifier: Apache-2.0
+
+"""Tests scikit-learn's SGDClassifier converter."""
+
+import sklearn
+import unittest
+import numpy as np
+import packaging.version as pv
+from sklearn.discriminant_analysis import QuadraticDiscriminantAnalysis
+from onnxruntime import __version__ as ort_version
+from onnx import __version__ as onnx_version
+from skl2onnx import convert_sklearn
+from skl2onnx.common.data_types import (
+    FloatTensorType,
+)
+
+from tests.test_utils import (
+    dump_data_and_model,
+    TARGET_OPSET
+)
+
+ort_version = ".".join(ort_version.split(".")[:2])
+onnx_version = ".".join(onnx_version.split('.')[:2])
+
+
+class TestQuadraticDiscriminantAnalysisConverter(unittest.TestCase):
+    @unittest.skipIf(pv.Version(sklearn.__version__) < pv.Version('1.0'),
+                     reason="scikit-learn<1.0")
+    @unittest.skipIf(pv.Version(onnx_version) < pv.Version('1.11'),
+                     reason="fails with onnx 1.10")
+    def test_model_qda_svm_2c2f(self):
+        # 2 classes, 2 features
+        X = np.array([[-1, -1], [-2, -1], [-3, -2], [1, 1], [2, 1], [3, 2]])
+        y = np.array([1, 1, 1, 2, 2, 2])
+        X_test = np.array([[-0.8, -1], [0.8, 1]])
+
+        skl_model = QuadraticDiscriminantAnalysis()
+        skl_model.fit(X, y)
+
+        onnx_model = convert_sklearn(
+            skl_model,
+            "scikit-learn QDA",
+            [("input", FloatTensorType([None, X.shape[1]]))],
```

Comment:
It is possible to add a unit test to make sure it is working with double?

Reply:
sure. let me add it.